crowbar_framework/app/helpers/barclamp/suse_manager_client_helper.rb

```diff
@@ -25,7 +25,7 @@ def suse_manager_client_rpm
     end
 
     def suse_manager_client_install
-      "/opt/dell/bin/barclamp_install.rb --rpm suse-manager-client"
+      "knife cookbook upload core -o /opt/dell/chef/cookbooks"
```

Comment:
But we do not need to upload all of them, we only need one file. See how the whole text looks like: http://crowbar.vi5.cloud.suse.de/crowbar/suse_manager_client/1.0/proposals/default